src/open_inwoner/templates/master.html

```diff
@@ -136,6 +136,13 @@
             </main>
         {% endblock main_outer %}
 
+        {% get_solo 'configurations.SiteConfiguration' as config %}
+        {% if config.kcm_survey_link_url and config.kcm_survey_link_text %}
+            <div class="kcm-survey">
+                {% link bold=True href=config.kcm_survey_link_url text=config.kcm_survey_link_text blank=True hide_external_icon=True secondary=True %}
+            </div>
+        {% endif %}
```

Comment:
I'm not entirely sure where to place to not mess with our tab order, I figured placing it before the footer made sense

Reply:
Alternatively: below the footer, since it is the only element that doesn't contain information relevant to the user; instead, it asks for the user's opinion about the entire site, so it would also make sense to put it at the end. However, your approach seems fine as well.

Reply:
I see on https://gemeente.groningen.nl/ they've indeed put it below the footer, right before the Body closes.
Currently we have the slightly similar floating element `<div class="anchor__scroll" id="scroll-anchor">` in Master above the Main section (because this used to be visible on Desktop) but I guess I'll have to move that one down in future also; so I would group these two together below the Footer (outside of the scope of this PR).